test/acceptance/keywords/content_assertions.robot

```diff
@@ -119,15 +119,20 @@ Page Should Not Contain Element With Disabling Source Logging
 
 Element Should Contain
     Element Should Contain    some_id    This text is inside an identified element
```

Comment:
Please add line: `| Element Should Contain | some_id | This text is inside an identified element | ignore_case=False |`